tests/Sniffs/NamingConventions/VariableLengthSniffTest.inc

```diff
@@ -18,6 +20,7 @@ class Test
         $red = 0;
         $green = 1;
         $bl = 2;
+        $em = 3;
```

Comment:
This should raise and error then. I'd keep it here and increase number of found errors.